samples/videoplayer/src/main/kotlin/Vec2D.kt

```diff
@@ -0,0 +1,5 @@
+
+data class Vec2D(val x: Int, val y: Int) {
```

Comment:
Vec2D -> Vector2D

Reply:
Renamed to `Dimensions(w,h)` based on the below comment.